app/poi_api/api.py

```diff
@@ -1,32 +1,32 @@
-from .poi_service import POI_Service
+from .service import list_POI, get_details_for_POI, new_POI_suggestion

```

Comment:
For the file naming you can even just use `app/poi/api`, we don't really need the `_api`. Or you can choose a different name beside `poi`, maybe locations?